package.json

```diff
@@ -102,23 +102,20 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "styled-components": "^4.4.1",
-    "ts-jest": "^25.4.0",
```

Comment:
No longer required since `@babel/preset-typescript` is doing the heavy lifting here.